src/app/app.module.ts

```diff
@@ -3,12 +3,20 @@ import { BrowserModule } from "@angular/platform-browser";
 import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
 
 import { LayoutModule } from "./layout/layout.module";
-
+import { AuthModule } from "./auth/auth.module";
+import { AppRoutes } from "./app-routing.module";
```

Comment:
according to names conversations `AppRoutes` should be followed by `module`.
pls. add space between modules and components.